src/System.Net.Primitives/src/System/Net/CookieCollection.cs

```diff
@@ -64,6 +65,7 @@ public void Add(Cookie cookie)
             {
                 throw new ArgumentNullException(nameof(cookie));
             }
+            ++m_version;
```

Comment:
But don't get me wrong, I totally understand that we don't really need the correct value in the field for compatibility with netfx. I'm just not sure if that's also true for earlier versions or other frameworks like mono.